### ISSUE TITLE
[Windows] Neovim can't find 'lspupdate' module after being installed

### ISSUE THREAD

**Issue by Creativty** [alexaandru/nvim-lspupdate#9]
Hi there, This might be a noob question; But why does this happen on my machine after running 
`:LspUpdate`

![image](https://user-images.githubusercontent.com/26884776/129962017-260fac24-1612-4206-aefe-5f470e646643.png)

I am running :
Windows 10
Neovim is 0.5.0 (Not Nightly)
`nvim-lspconfig` is installed.
I use neovim's Packer, Not Plug.
I have the package managers installed. (npm, pip)

--Edit: I did notice that the package itself that it's looking for exists under `%LOCALAPPDATA%/nvim-data/site/pack/packer/start/nvim-lspupdate/`
Although the content looks like it is in Fennel (as in not similair to other packages I have installed), Which I have never interacted with before :/

Sorry for the nuissance.
Hope you have a good day :)

**Comment by alexaandru** [comment]
Hello,

If you check out the README, there is a note there about some recent changes to the project, as well as the proposed solutions: either install an additional plugin (Hotpot) or run `make lua` (but you will need Lua on your system).

With **packer**, provided that **Lua** is available, should be as easy as:

```
use {'alexaandru/nvim-lspupdate', run = 'make lua'}
```

Hope this helps &
you too have a good day :) 

Cheers!

**Comment by Creativty** [comment]
Thanks for the swift response.
I did understand the options you provided, however I opt to not install Fennel (sadly, I barely know lua so...)
When I ran with `use {'alexaandru/nvim-lspupdate', run = 'make lua'}` It just keeps getting stuck at `make lua part`. and I think I understand why, since windows doesn't have Make, I replaced the command `make lua` with `mingw32-make lua` however that also didn't work. So I decided to go the package folder and run the command manually.
But it results with this : 
![image](https://user-images.githubusercontent.com/26884776/129971051-8b43509a-8bbe-418f-a12c-8e9705b36852.png)

And I think this is because the make script was made with linux in mind. (Reminder: I do have lua installed, and it's in my path as seen in the image)
Thanks in advance.

**Comment by alexaandru** [comment]
Hmm, looks like `env` cannot find `lua` for some reason.

I just pushed a change (https://github.com/alexaandru/nvim-lspupdate/commit/0a4d3f3f9125955bab3245009973a179b8ba5538) could you please update the plugin and try again? This should get you past that error, as it gets `env` out of the equation.

**Comment by Creativty** [comment]
So yeah, I managed to fix it. However only after certain complications. Because even after the change you performed, There was still linux considerations in the Makefile, "Like `mkdir -p`, `gunzip...`chmod`..." However luckily I did have Git Bash, Which simulates a linux-terminal like environment on Windows. So I ran the `mingw32-make lua` manually throught Git and it compiled. Now `:LspUpdate` works.
Thanks for your time and consideration :D
